cppcache/integration/test/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ add_executable(cpp-integration-test
   ExpirationTest.cpp
   FunctionExecutionTest.cpp
   PdxInstanceTest.cpp
+  PdxJsonHandlerTest.cpp
```

Comment:
Where is the term "handler" coming from in this test name? The thing you are testing is PDX encode JSON via  the `PdxInstance` and `PdxType` class. You aren't registering a new `PdxTypeHandler` so the use of handler term is confusing me.

Reply:
Yep, started investigating type handler on this branch, then fixed this bug and never changed the terminology.